paasta_tools/utils.py

```diff
@@ -143,6 +146,18 @@ def get_cpus(self):
         cpus = self.config_dict.get('cpus', .25)
         return cpus
 
+    def get_cpu_period(self):
+        return self.config_dict.get('cfs_period_us', DEFAULT_CPU_PERIOD)
+
+    def get_cpu_quota(self):
+        cpu_burst_pct = self.config_dict.get('cpu_burst_pct', DEFAULT_CPU_BURST_PCT)
+        return self.get_cpus() * self.get_cpu_period() * (100 + cpu_burst_pct) / 100
+
+    def get_docker_parameters(self):
```

Comment:
Actually, can we call this "format_docker_parameters", since this isn't a true getter, and add a docstring explaining what it's doing and what format it's in?